pages/common/topgrade.md

```diff
@@ -0,0 +1,28 @@
+# Topgrade
+
+> Update all your applications.
+> More at <https://github.com/r-darwish/topgrade>.
+
+- Call updates:
+
+`topgrade`
+
+- Say yes to all updates:
+
+`topgrade -y`
+
+- Cleanup temporary/old files:
+
+`topgrade -c`
+
+- Disable a certain step:
```

Comment:
Can we explain what is a "step"?

Reply:
I have replaced it with "update operation"